homeassistant/helpers/selector.py

```diff
@@ -299,8 +300,9 @@ class ColorTempSelector(Selector[ColorTempSelectorConfig]):
 
     CONFIG_SCHEMA = vol.Schema(
         {
-            vol.Optional("max_mireds"): vol.Coerce(int),
-            vol.Optional("min_mireds"): vol.Coerce(int),
```

Comment:
This is breaking any blueprint out there right now, we cannot do that.

Reply:
Did not think of blueprints 🤦 So we keep the old params and add new "min" and "max" to be used for Kelvin and potentially at some point in the future phase out the old ones with "mired" in the name?